apps/glusterfs/volume_entry_create.go

```diff
@@ -114,7 +114,14 @@ func (v *VolumeEntry) createVolumeRequest(db wdb.RODB,
 	// Setup volume information in the request
 	vr.Name = v.Info.Name
 	v.Durability.SetExecutorVolumeRequest(vr)
-	vr.GlusterVolumeOptions = v.GlusterVolumeOptions
+	// filter out undesired options from gluster (TOO HACKY?)
+	vr.GlusterVolumeOptions = []string{}
+	for _, s := range v.GlusterVolumeOptions {
```

Comment:
Let us not filter out the arbiter option. It would be really helpful in debugging if gluster stored what heketi knew about the volume. As it is user.heketi. it is harmless.

Reply:
Sure. I will make that change.